server/services/interventionsService.test.ts

```diff
@@ -2564,39 +2565,67 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 
-  describe('getSupplierAssessmentAppointment', () => {
-    const appointment = appointmentFactory.build({
-      appointmentTime: '2021-05-13T12:30:00Z',
-      durationInMinutes: 120,
-    })
+  describe('getSupplierAssessment', () => {
```

Comment:
Sorry, not sure how to leave this comment on the commit message, but should it say:

>  Add contract for fetching the supplier assessment 
> This contains any supplier assessment **appointments** that have been booked, and hence
> replaces the existing getSupplierAssessmentAppointment method.

Reply:
Ah, thanks for spotting that 🙂